epictrack-api/src/api/reports/anticipated_schedule_report.py

```diff
@@ -150,34 +93,27 @@ def _fetch_data(self, report_date):
             .join(Region, Region.id == Project.region_id_env)
             .join(EAAct, EAAct.id == Work.ea_act_id)
             .join(Ministry)
+            .join(latest_status_updates, latest_status_updates.c.work_id == Work.id)
```

Comment:
Have we discussed joining it with status update query. Cz I think as per the latest logic, the result has nothing to do with the status update